modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -15,75 +15,219 @@ Additional detail on how Red Hat provides support for bundled community dynamic
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-ocm-backend-dynamic
+
+`OCM_HUB_NAME`
+
+`OCM_HUB_URL`
+
+`moc_infra_token`
+
+|Disabled
+
+|Quay  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-quay/v/1.8.0[@janus-idp/backstage-plugin-quay] |1.8.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-quay
+
+|Disabled
+
+|Quay  - Backend |https://npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-quay/v/1.5.0[@janus-idp/backstage-scaffolder-backend-module-quay] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-quay-dynamic
+
+|Enabled
+
+|RBAC  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-rbac/v/1.23.2[@janus-idp/backstage-plugin-rbac] |1.23.2 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-rbac
+
+|Disabled
+
+|Regex  - Backend |https://npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-regex/v/1.5.0[@janus-idp/backstage-scaffolder-backend-module-regex] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-regex-dynamic
+
+|Enabled
+
+|TechDocs  - Frontend |https://npmjs.com/package/@backstage/plugin-techdocs/v/1.10.5[@backstage/plugin-techdocs] |1.10.5 
+|./dynamic-plugins/dist/backstage-plugin-techdocs
+
+|Enabled
+
+|TechDocs  - Backend |https://npmjs.com/package/@backstage/plugin-techdocs-backend/v/1.10.5[@backstage/plugin-techdocs-backend] |1.10.5 
+|./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic
+
+|Enabled
+
+|Tekton  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-tekton/v/3.8.0[@janus-idp/backstage-plugin-tekton] |3.8.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton
+
+|Disabled
+
+|Topology  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-topology/v/1.22.0[@janus-idp/backstage-plugin-topology] |1.22.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-topology
+
+|Disabled
+
+|===
+[NOTE]
+====
+* To configure Keycloak, see xref:rhdh-keycloak_{context}[Installation and configuration of Keycloak].
+
+* To configure Techdocs, see http://backstage.io/docs/features/techdocs/configuration[reference documentation]. After experimenting with basic setup, use CI/CD to generate docs and an external cloud storage when deploying TechDocs for production use-case.
+See also this https://backstage.io/docs/features/techdocs/how-to-guides#how-to-migrate-from-techdocs-basic-to-recommended-deployment-approach[recommended deployment approach].
+====
+
+== Technology Preview plugins
+
+=== Red Hat plugins
+
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
This information must be included as a Note.

Reply:
It used to be a note, but in https://docs.google.com/document/d/1F6ctaXcUis4ngcxGYhdPmdY23iQzQgz1v4lObn6WqZc/edit#heading=h.60hf0c6z3pg4 it was moved to a paragraph header by Gerry.

Reply:
The TP note/admonition preceding the _Red Hat plugins_ and _Community plugins_ headers should be notes - I've updated the Gdoc to reflect this.